components/shell/CommandPalette/actions.ts

```diff
@@ -13,7 +13,6 @@ export const getStaticActions = ({
   {
     id: 'quick-action-modal',
     name: 'Open Quick Action Panel',
-    shortcut: ['q'],
```

Comment:
we never use these global shortcuts and they break some other shortcuts we use contextually in various panels so getting rid of these entirely.
fixes https://github.com/bluesky-social/ozone/issues/265